app/QUICKSTART.md

```diff
@@ -6,9 +6,9 @@ The `main` function in `Main.hs` does two things:
 * Serialise the instantiated validator and write it to a file
 
```

Comment:
@zliu41 This entire file must be revisited as it is now obsolete.

Reply:
Once https://github.com/IntersectMBO/plutus/pull/6477 is merged, I'll remove most content of this file, and point to the docusaurus site instead.